### PR TITLE
use new button component

### PR DESCRIPTION
**Pull request by vanpertsch**
**Before**
Screenshot 2022-02-28 at 11 13 42

**After**

Screenshot 2022-02-28 at 11 44 08

### CODE REVIEW

File: apps/settings/src/components/UserList.vue [hunk 1]
```diff
@@ -142,12 +142,11 @@
 				<div v-if="showConfig.showUserBackend" class="userBackend" />
 				<div v-if="showConfig.showLastLogin" class="lastLogin" />
 				<div class="user-actions">
-					<button id="newsubmit"
-						class="button primary"
-						type="submit"
+					<Button id="newsubmit"
+						type="submit primary"
```
Comment (marcoambrosini) [inline apps/settings/src/components/UserList.vue:146]:
```suggestion
						type="primary"
						nativeType="submit"
```

**Review by jancborchardt — APPROVED** [review]
Screenshot confused a bit since it looks non-primary cause it's focused, but code looks good. :)

**Review by skjnldsv — APPROVED** [review]

**Review by marcoambrosini — APPROVED** [review]

**Pull request merged**